README.md

```diff
@@ -1,6 +1,18 @@
 # Beepo
 
-A basic morse code trainer intended for experimentation
+A basic morse code trainer intended for experimentation.
+
+Features:
+
+* Memorise morse code by progressive learning similar to Koch method
+* Adjustable parameters
+* Timing
```

Comment:
A more elaborate explanation of this feature would be helpful.

Reply:
Thanks, I plan to improve that as the program gets a bit more polished. This is more a statement of intent and subject to change in the future.